extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -108,15 +119,53 @@ export class AmpAudio extends AMP.BaseElement {
   pauseCallback() {
     if (this.audio_) {
       this.audio_.pause();
+      if (getMode().test) {
+        this.isPlaying_ = false;
+      }
+    }
+  }
+
+  pause() {
+    const storyEl = closestByTag(this.element, 'AMP-STORY');
+    if (this.audio_ && !storyEl) {
```

Comment:
This should be storyEl != parentElement?

Reply:
Also do I care about `amp-story` or `amp-story-page`?

Reply:
So just to clarify, if this is the child of a `<amp-story>` element, pause should do nothing? Can we add a comment explaining this to devs not familiar with `<amp-story>`?

Reply:
Good catch! Both play and pause actions should do nothing. Added a comment in latest commit.